lib/vrf.c

```diff
@@ -20,6 +20,7 @@
  */
 
 #include <zebra.h>
+#include "zebra/zebra_vrf.h"
```

Comment:
This is not right. We can't include zebra-specific headers in the library file.

Reply:
Thank you for catching this. I've removed it in the most recent commit.